gilded_rose.py

```diff
@@ -1,42 +1,26 @@
 # -*- coding: utf-8 -*-
+from collections import defaultdict
+
+from updaters import (AgedBrieUpdater, BackstagePassesUpdater, ConjuredUpdater,
+                      RegularUpdater, SulfurasUpdater)
 
 
 class GildedRose(object):
     def __init__(self, items):
         self.items = items
+        self.updaters = defaultdict(
```

Comment:
Really great that you've tried to use `defaultdict` (so u know that it exist), however I would rather stick to ordinary python dict and simply use `self.updaters.get(item.name, RegularUpdater.tick)(item)`. It's because it does exactly the same and there is no extra import necessary